core/arch/opnd.h

```diff
@@ -300,6 +300,7 @@ enum {
     DR_REG_CR13,
     DR_REG_CR14,
     DR_REG_CR15,
+    DR_REG_MAX_AS_OPSZ = DR_REG_CR15,
     DR_REG_INVALID, /**< Sentinel value indicating an invalid register. */
```

Comment:
(I suppose there's no simple way to ensure larger DR_REG values are not used in the decode tables.)

Reply:
Indeed. The comment mentions it.